code/clue.js

```diff
@@ -1,59 +1,347 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+//LAYOUT MODIFICATIONS ON PAGE LOAD
+
+document.getElementById('killerCard').classList.add("active");
+document.getElementById('weaponCard').classList.add("active");
+document.getElementById('roomCard').classList.add("active");
```

Comment:
Since you add these classes at the very beginning without any user trigger, they should be just written in HTML